### PR TITLE
1_description: Add DESCRIPTION and NAMESPACE

### LINKED ISSUE #1: Feature Request:  Add DESCRIPTION
### Feature Idea

Add `DESCRIPTION` file

### Relevant Input

What should the input look like? REMINDER: no patient level data or company sensitive information should be shared via this open public issue

### Relevant Output

What should the output look like? REMINDER: no patient level data or company sensitive information should be shared via this open public issue

### Reproducible Example/Pseudo Code

We love example code and it will speed up the process! REMINDER: no patient level data or company sensitive information should be shared via this open public issue

### PR DESCRIPTION
**Pull request by bundfussr**
Closes #1 .

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiralonco/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Package: admiralonco
+Type: Package
+Title: Oncology extension package for ADaM in R Asset Library 
+Version: 0.0.0
+Authors@R: c(
+    person("Stefan", "Bundfuss", email = "stefan.bundfuss@roche.com", role = c("aut", "cre")),
+    person("Ross", "Farrugia", role = "aut"),
+    person("Amit", "Jain", role = "aut"),
+    person("Liming (Catherine)", "Jin", role = "aut"),
```
Comment (rossfarrugia) [inline DESCRIPTION:9]: Add other authors as: 
Andrew Smith
Cloris Xue 
Caihong Xia 
Karl Kennedy
Konstantina Koukourikou 
Mary Dee
Neharika Sharma 
Samia Kabi
Stephen Gormley
Annie Yang 

And contributors as: 
Gillian Yin
Aiming Yang
Gopi Vegesna

File: DESCRIPTION [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    person("F. Hoffmann-La Roche AG", role = c("cph", "fnd")),
+    person("GlaxoSmithKline LLC", role = c("cph", "fnd")),
+    person("Amgen Inc.", role = c("cph", "fnd"))
+    )
+Description: A toolbox for programming oncology specific Clinical Data Standards Interchange Consortium
```
Comment (rossfarrugia) [inline DESCRIPTION:14]: Should we also add mention of this being a package extension of {admiral}? - probably obvious by the name but can't harm

File: DESCRIPTION [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    implemented in accordance with the "Analysis Data Model Implementation Guide"
+    (CDISC Analysis Data Model Team, 2021, <https://www.cdisc.org/standards/foundational/adam/adamig-v1-3-release-package>).
+License: Apache License (>= 2)
+Encoding: UTF-8
+LazyData: true
+Roxygen: list(markdown = TRUE)
+RoxygenNote: 7.1.2
+Depends: R (>= 3.5)
+Imports:
+    admiral,
```
Comment (rossfarrugia) [inline DESCRIPTION:27]: add admiral.test for RS/TU/TR test data and could consider adding all the others admiral uses as chances are we'll need similar? or maybe add them as needed over time if you prefer

**Review by rossfarrugia — CHANGES_REQUESTED** [review]
Thanks for starting this off @bundfussr - i left some minor review comments

File: DESCRIPTION (hunk 1 above)
Comment (bundfussr) [reply DESCRIPTION:9]: Added

File: DESCRIPTION (hunk 2 above)
Comment (bundfussr) [reply DESCRIPTION:27]: Agree and added

File: DESCRIPTION (hunk 3 above)
Comment (bundfussr) [reply DESCRIPTION:42]: I have added admiral.test. I would add others when needed. That's easier than checking later which ones are not needed.

**Comment by rossfarrugia** [comment]
@bundfussr reminder to run devtools::document() to update man/admiralonco-package.Rd and looks like some checks failing to resolve

**Comment by bundfussr** [comment]
HI @kabis-ops , from my side we can merge it. I requested re-review from Ross because he requested changes.

**Review by rossfarrugia — APPROVED** [review]
Happy to merge and we can always go back to the contributor list later to update before we make a release in main, as new people have joined since i believe - hard to keep up across all these packages.

**Comment by rossfarrugia** [comment]
@kabis-ops over to you for building the site - thanks!

**Pull request merged**